### ISSUE TITLE
Footnote on list item formatted incorrectly

### ISSUE THREAD

**Issue by aridus** [grimbough/msmbstyle#8]
With msmbstyle, if one adds a footnote to a list item the footnote in the right hand column is aligned too far to the left (i.e. it does not left align with other items in the column).

This code illustrates the problem:

`- Literature citations do not have matching references, and vice versa.^[_Solution:_ use a [reference manager][Reference management].]

Literature citations and references are not in the journal's preferred format.^[_Solution:_ use a [reference manager][Reference management].] `

Would it be possible for this to be fixed?

With grateful thanks!

**Comment by grimbough** [comment]
I think I've sorted this in the CSS file now.  Try installing version 0.0.7 and see if the update works for you too.

**Comment by aridus** [comment]
Many thanks for being responsive! I have installed 0.0.7. It has caused a change but perhaps not that expected. I have shared three screenshots at

https://www.dropbox.com/sh/4rb2grhr1l0z19j/AACLYfqSEkvOoh-TF7bUhC4Fa?dl=0

Screenshot_zero shows the footnotes on a list

Screenshot_one shows the footnotes on a blockquote

Screenshot_two shows that a piece of text on {marginfigure}, which formally was correctly aligned left, is now in the wrong position.

I would be happy to test further to help you resolve  this  issue.

With warm regards.

**Comment by grimbough** [comment]
Thanks for the screen shots.  I wouldn't have expected the block quote to change as I only modified the CSS for margin notes inside lists, but it shouldn't have broken the 'standard' margin text.

Any chance you can put the examples in a single Rmd file for me to compile? Then I can make sure I'm constructing them in the same way you are.

**Comment by aridus** [comment]
Thank you: this seems like a good idea, in case I am doing anything silly - always a possibility. I have uploaded to the same folder 'structure (copy).Rmd'. I have deleted graphics, so that it should compile fine (but let me know if not).

Thanks again.

**Comment by grimbough** [comment]
Thanks for that.  I hadn't really defined the `blockquote` environment at all, so it was just falling back on whatever the browser default is.  I think this is sorted now, so feel free to give it a test.

**Comment by aridus** [comment]
Thank you, again, and all is now well!

As an aside, could the blockquote environment benefit from better definition? At present the text is simply indented (there is no grey background or other indication). 

In the bookdown manual at https://bookdown.org/yihui/bookdown/markdown-syntax.html, below the text 'The actual output (we customized the style for blockquotes in this book):' a blockquote has a light grey bar on the left-hand side, and if one narrows the page scrollbars do not appear (as they do on a code chunk). I think this behaviour (light grey bar, no scroll bars) works very well and would like to see it in msmbstyle.

I am using msmbstyle to prepare two instruction manuals, and it is working well, so thank you for your efforts. The manuals will go live sometime in April. I can send you a link if you wish.

The only matter I am struggling with is how to tailor the top bar and menu so that these items fit in with our website (but this is down to my extremely rudimentary knowledge of css).

Yours, etc.

**Comment by grimbough** [comment]
Yes, I agree.  At the moment **msmbstyle** is just piggy-backing on the **tufte** package, and that seems to define block quote as a simple indentation, which doesn't seem in the spirit of how blockquote is supposed to be used.  I'll have a look at some examples and maybe refine it a bit, but I'll try to keep the side notes inline when I do!

**Comment by aridus** [comment]
Many thanks, and I look forward to news!

Bookdown + tufte style is an unbeatable combination for a teaching tool, and msmbstyle provides several improvements over the original bookdown implementation, hence my enthusiasm for it.

I'm sure I will have other questions. I will be in touch.

Yours, etc.

**Comment by aridus** [comment]
Just to let you know: marginal footnotes on a list are still slightly out of line. I see this in the following example: 

`There are a few cases in which it is reasonable to use acronyms:

1. Some are now widely known---and often used in preference to the full name---that it seems reasonable to use them, but indicating their expansion at first use^[e.g. CBD for [Convention on Biological Diversity](https://www.cbd.int){target="_blank"}]. As most readers will have encountered the acronym in other writing, the first use reminds them and they will not be burdened by its repetition in the text.

2. Some are both widely known and used in everyday speech and writing to the extent that they function almost as replacement names^[e.g. IUCN for [International Union for Conservation of Nature](https://www.iucn.org){target="_blank"}]. In such rare cases, the expansion of the acronym is unnecessary even at first use.

By convention, abbreviations and acronyms are always expanded at the begninng of a sentence, even if they have been used earlier in the text, as here:`

You can also see the problem at 6.3 in https://www-huber.embl.de/msmb/Chap-Testing.html

Thanks for your attention.

**Comment by aridus** [comment]
Hi - This is just to ask whether you have any further thoughts  on this.

With grateful thanks!

**Comment by grimbough** [comment]
Is this still an issue?  To my eyes it looks fine, but I'm happy to believe it it's some thing that works with my screen resolution/browser/fonts combo, but doesn't for you.

Here's a screen shot with a file browser put on top so I can check the alignment. The first footnote is from a normal paragraph, the bottom two in a numbered list:

![Peek 2019-04-29 14-58](https://user-images.githubusercontent.com/971237/56897666-693d3f80-6a8f-11e9-9c17-30889bc620d9.gif)

Similarly, if I scroll through https://www-huber.embl.de/msmb/Chap-Testing.html the footnotes in 6.3 seem to align with all the rest.  Perhaps you can screen shot it too?

**Comment by aridus** [comment]
![Screenshot from 2019-04-29 17-22-16](https://user-images.githubusercontent.com/11365547/56910970-b9260180-6aa3-11e9-9650-2f9e734995ab.png)

Many thanks for looking at this again. I have just installed 0.0.8 and recompiled my book (see attached screenshot). Footnotes 42 and 43, on the bulleted list are slightly to the left of footnote 44, which is on normal body text.

With thanks!
